x-pack/plugins/secops/server/lib/uncommon_processes/types.ts

```diff
@@ -76,3 +73,25 @@ export interface UncommonProcessData extends SearchHit {
     };
   };
 }
+
+export interface HostHits {
```

Comment:
That's also a duplicate from `server/lib/hosts/types`, when we see it we should fix it or we not going to know which one to chose.
 I think it will pretty easy to export from `server/lib/types` and you can re-import from there

Reply:
It kind of _is_ and kind of is not. I can try to reconcile between them but the types can quickly diverge since a lot of Elastic types of what it returns is dependent on what is sent in.

For example, looking at this definition we have:

```ts
export interface HostBucket {
  key: { host_name: string };
  host: {
    hits: {
      total: number;
      max_score: number | null;
      hits: HostHit[];
    };
  };
}
```

That key is very different from possibly another key because it is defined with the agg terms in the codebase from here:

```
sources: [{ host_name: { terms: { field: 'host.id' } } }],
```

Where my key is a direct string.

Also either we'll end up with a lot of optionals since we're using `top_hits` with `_source` to selectively get fields between queries such as this one:
```ts
top_hits: {
  size: 1,
  _source: esFields
}

or we will end up using a lot of `Pick` and/or `Omit` with types. However, I will try to reconcile and make these work closer together.